files/grest/rpc/assets/asset_policy_info.sql

```diff
@@ -87,9 +88,11 @@ BEGIN
       mtm.metadata as minting_tx_metadata,
       trm.metadata as token_registry_metadata,
       ts.amount::text as total_supply,
```

Comment:
Sort of same argument as asset_info vs asset_info_bulk for asset_policy_info vs asset_policy_list. I dont think we should keep both of these, but only one. To me asset_policy_info is too spammy and contain data best left for asset_info. 

I would like to replace(can think about what name to keep):
asset_info -> asset_info_bulk
asset_policy_info -> asset_policy_list

And then instead just settle on exactly what we want to include in each.

Reply:
Changing total_supply value from CTE join to cache table shouldn't be breaking tho

Also, the breaking part will be if we're retiring old URLs, rather than mark them deprecated and add new one (for asset_info) while for asset_policy_info, it doesnt do much harm to leave , as it'll expire in 30s if a spammy asset - instance runners can customise that on their ends in that case

Reply:
ok, but if we want to modify it and keep content I can make it more efficient than just switching out this CTE join.